util/symbolize/dwarf3ranges.cc

```diff
@@ -64,10 +64,10 @@ void AddressRangeList::ReadDwarfRangeList(uint64 offset, uint64 base,
 void AddressRangeList::ReadDwarfRngListsDirectly(uint64 offset, uint64 base,
                                             AddressRangeList::RangeList* ranges, uint64 addr_base) {
   // Read rnglists from .debug_rnglists when the format was DW_FORM_sec_offset.
-  // Since we are calculating the address with DW_FORM_sec_offset, 
-  //  we don't care about (or need) the index to rngdatamap_ here. 
+  // Since we are calculating the address with DW_FORM_sec_offset,
+  //  we don't care about (or need) the index to rngdatamap_ here.
   //  Therefore, we are getting the first element (rngdatamap_.begin())
-  //  and getting the address_size from that element. 
+  //  and getting the address_size from that element.
```

Comment:
Sorry, I guess my comment wasn't clear. You have extra space at the beginning of the each of the last 3 lines.